spec/requests/cinema_halls_request_rspec.rb

```diff
@@ -4,41 +4,83 @@
 
 RSpec.describe 'CinemaHalls', type: :request do
   describe 'GET /cinema_halls' do
+    let!(:cinema_hall) { CinemaHall.create(name: 'Lalka', number_of_seats: 100) }
```

Comment:
There are two different variables: `let` and `!let`. We prefer to use the first one without exclamation mark because during creating a lot of tests we would like to have specific records in every `context`. Creating variables with `let` you need to create them in tasted environment with `before`. 

```ruby
  context "GET cinema_halls/" do
    let(:cinema_hall) { ... }
    
    before do
      cinema_hall
    end
    
    it "returns cinema halls" do
      ... 
    end
  end
  ```
  
Try creating some `cinema_hall` at the top of the `describe` (under line 5) and one with `let` and another with `!let` and look at the difference using `byebug` what records of `CinemaHall` you have :)